cw/cw305/simple_capture_traces.py

```diff
@@ -95,6 +107,10 @@ def parse_args():
                       '-p',
                       type=int,
                       help="Plot number of traces.")
+  parser.add_argument('--start-delay',
```

Comment:
I think you can drop changes related to start delay since we will be using SAM3U for programming.

Reply:
Yep, I agree. Dropped the corresponding commit.